pha-admin/src/main/java/com/store/pharmacy/Category/repository/CategoryRepository.java

```diff
@@ -3,8 +3,10 @@
 import org.springframework.data.jpa.repository.JpaRepository;
 
 import com.store.pharmacy.Category.model.Category;
```

Comment:
Rename this package name to match the regular expression ^[a-z_]+(.[a-z_][a-z0-9_])$